app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/completeregistration/CompleteRegistrationViewModel.kt

```diff
@@ -259,6 +257,36 @@ class CompleteRegistrationViewModel @Inject constructor(
         }
     }
 
+    private fun handleLoginAttempt(action: Internal.AttemptLogin) {
+        viewModelScope.launch {
+            val result = authRepository.login(
+                email = state.userEmail,
+                password = state.passwordInput,
+                captchaToken = action.captchaToken,
+            )
+            sendAction(Internal.ReceiveLoginResult(result, action.captchaToken))
```

Comment:
Idk how much you guys are into naming params on this project, and these are pretty self-explanatory, but option to do so if you're into it